index.ts

```diff
@@ -1094,6 +1110,33 @@ commander
     });
   });
 
+/**
+ * Clasp run <function>
+ * 
+ * @param scriptId ID of the script you want to run
+ * @param theFunction function in the script that you want to run
+ * 
+ * Note: to use this command, you must have used `clasp login --localkey`
+ */
+commander
+  .command('run <scriptId> <theFunction>')
```

Comment:
- Always default to the current script.
  - `.command('run <theFunction>')`
- Use `functionName` instead of `theFunction`